app/views/avo/partials/_flash_alerts.turbo_stream.erb

```diff
@@ -1,3 +1,3 @@
 <%= turbo_stream.append "alerts" do %>
-  <%= render Avo::FlashAlertsComponent.new flashes: flash %>
+  <%= render Avo::FlashAlertsComponent.new flashes: flash.discard %>
```

Comment:
Are we sure that the `discard` works well with other flows as well?

Reply:
I think so, it will discard the entire flash at the end of the current action